media/js/firefox/firstrun/firstrun_iframeless_fxa.js

```diff
@@ -65,4 +65,23 @@
         }
     };
 
+    fetch('http://127.0.0.1:3030/metrics-flow').then((resp) => {
```

Comment:
@jpetto as part of this get request could you please pass any query params the pages gets (such as `utm` params, also maybe A/B variations, `entrypoint`, etc)